Week-4/assignment-1.js

```diff
@@ -0,0 +1,122 @@
+const STATE = {
+  FULFILLED: "fulfilled",
+  REJECTED: "rejected",
+  PENDING: "pending"
+};
+class Promise {
```

Comment:
lets use some name so that we dont have conflict while reading

Reply:
Done...